packages/flutter/lib/src/material/icons.dart

```diff
@@ -34,10 +55,12 @@ class Icons {
   // BEGIN GENERATED
 
   /// <i class="material-icons md-36">360</i> &#x2014; material icon named "360".
-  static const IconData threesixty = IconData(0xe577, fontFamily: 'MaterialIcons');
+  static const IconData threesixty =
```

Comment:
You should revert all of these changes back to one-liners, since they're not related to your current PR